tests/src/Unit/OgAdminRoutesControllerTest.php

```diff
@@ -202,11 +214,28 @@ public function testRoutesWithAccess() {
 
   }
 
+  /**
+   * Tests overview with hidden routes.
+   *
+   * @covers ::overview
+   */
+  public function testHiddenOverviewRoutesAreFiltered() {
+    $result = $this->getRenderElementResult(TRUE);
+    // Filter all hidden routes.
+    $hidden_routes_info = array_filter($this->routesInfo, function($route) {
```

Comment:
Expected 1 space after FUNCTION keyword:

```suggestion
    $hidden_routes_info = array_filter($this->routesInfo, function ($route) {
```